### DISCUSSION TITLE
Opening freeplane from taskbar creates second taskbar icon

### DISCUSSION THREAD

**Discussion by la-moreno** [freeplane/freeplane#1882] [Help] [answered]
Hello, so this is not a major problem but I thought I would ask anyway.

I've noticed that, although you can pin the freeplane executable to the taskbar, when you launch freeplane it creates a second icon on the taskbar which does not merge with the pinned icon. Does anyone know why this is and if there's a fix?

Here's a picture for reference:
![image](https://github.com/freeplane/freeplane/assets/47702497/d526c006-d0d6-4735-a0b3-5abbf96c34dc)

**Comment by dpolivaev** [comment, marked as answer]
Known issue, I have no fix for it.

**Reply by walmar** [reply]
The following linked page seems to have some suggestions on how to prevent a Java application from starting as javaw.exe with its own taskbar icon without jump list items:
[Pinning a Java executable (with launch4j) to the Windows 7 taskbar - Stack Overflow](https://stackoverflow.com/questions/9910587/pinning-a-java-executable-with-launch4j-to-the-windows-7-taskbar)
